### PR TITLE
Add chunk manager request timeout

### PR DESCRIPTION
**Pull request by zhagnlu**
#27587

### CODE REVIEW

File: pkg/util/paramtable/base_table.go [hunk 1]
```diff
@@ -51,6 +51,7 @@ const (
 	DefaultKnowhereThreadPoolNumRatioInBuild = 1
 	DefaultMinioRegion                       = ""
 	DefaultMinioUseVirtualHost               = "false"
+	DefaultMinioRequestTimeout               = "3000"
```
Comment (czs007) [inline pkg/util/paramtable/base_table.go:54]: DefaultMinioRequestTimeoutMs?

Comment (czs007) [reply pkg/util/paramtable/base_table.go:54]: Increase the default timeout duration here? For example, to around 30 seconds?

File: configs/milvus.yaml [hunk 2]
```diff
@@ -94,6 +94,8 @@ minio:
   region: ""
   # Cloud whether use virtual host bucket mode
   useVirtualHost: false
+  # timeout for request time in milliseconds
+  requestTimeoutMs: 3000
```
Comment (czs007) [inline configs/milvus.yaml:98]: How about changing the unit of the timeout duration to seconds?

Comment (zhagnlu) [reply configs/milvus.yaml:98]: sdk default value is 3000 millseconds, and keep this default is better, sdk will retry for 10 times.  change to seconds seems no big advantage and millseonds is more fine-grained.

**Comment by zhagnlu** [comment]
rerun ut

**Pull request merged**